elements/lisk-p2p/src/p2p.ts

```diff
@@ -983,39 +982,49 @@ export class P2P extends EventEmitter {
 		return !!isSeed || !!isWhitelisted || !!isFixed;
 	}
 
-	public async start(): Promise<void> {
-		if (this._isActive) {
-			throw new Error('Cannot start the node because it is already active');
-		}
+	private _initializePeerBook(): void {
+		const newPeersToAdd = [
+			...this._sanitizedPeerLists.previousPeers,
+			...this._sanitizedPeerLists.whitelisted,
```

Comment:
why no `fixedPeer here`?

Reply:
They could, but we are connecting to fixedPeers are not affected by having them in PeerBook. [PeerPool](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-p2p/src/peer_pool.ts#L432)

Reply:
Implemented